packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -670,7 +670,7 @@ class SliverChildListDelegate extends SliverChildDelegate {
       return null;
     Widget child = children[index];
     final Key key = child.key != null? _SaltedValueKey(child.key) : null;
-    assert(child != null);
+    assert(child != null, "The sliver's children must not contain null values, but a null value was found at index $index");
```

Comment:
Break this line of code into a couple of lines. We want it to be <80 chars per line if possible